src/sqlfluff/rules/std.py

```diff
@@ -337,17 +337,24 @@ def _eval(self, segment, raw_stack, memory, **kwargs):
                 if this_line['indent_size'] != res[k]['indent_size']:
                     # Indents don't match even though balance is the same...
                     memory['problem_lines'].append(this_line_no)
+                    print(this_line_no, k, this_line['indent_balance'], res[k]['indent_balance'])
```

Comment:
Did you mean to leave the `print()` statements here and below?

Reply:
Ha! Nope I'll pull that one out. Good catch. Debugging this one was hard 🤕